machine-learning/tutorials/GitHubIssueClassification.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
given you have one project, I'd probably not add the sln file to avoid outdated vs versions quickly

Reply:
The others have .sln files so I'll keep it but I updated the version.